go.mod

```
@@ -1,8 +1,8 @@
 module github.com/getsops/sops/v3
 
-go 1.22
+go 1.22.7
```

Please keep the current values for `go` and `toolchain`.
```suggestion
go 1.22
```